bay/images/Dockerfile.php

```
@@ -1,5 +1,8 @@
 FROM amazeeio/php:7.2-fpm
 
+# Disable calling phpinfo().
+RUN sed -i 's/disable_functions =/disable_functions = phpinfo,/' /usr/local/etc/php/php.ini
```

@nicksantamaria , can we expand this, a lot of these are only required by drupal and drupal console so they could be limited to the cli container only. What do you think?

```suggestion
RUN sed -i 's/disable_functions =/disable_functions = exec,passthru,shell_exec,system,proc_open,popen,curl_exec,curl_multi_exec,parse_ini_file,show_source,phpinfo,/' /usr/local/etc/php/php.ini
```

---

I like the idea. The only ones that bother me are the `curl` ones... will this break `drush rsync`? The full list recommended by OWASP for reference:

```
disable_functions       = system, exec, shell_exec, passthru, phpinfo, show_source, highlight_file, popen, proc_open, fopen_with_path, dbmopen, dbase_open, putenv, move_uploaded_file, chdir, mkdir, rmdir, chmod, rename, filepro, filepro_rowcount, filepro_retrieve, posix_mkfifo
```

---

It would be great to have the QA time to actually test out the full OWASP list, then pull out the ones that cause issues.

---

@nicksantamaria / @anthony-malkoun , tracking the next step here, https://digital-engagement.atlassian.net/browse/SDPA-4557

---

@kurtfoster can you double-check that jira link? That one is "Adding a Twitter Social Share Image"